src/main/resources/application.yml

```diff
@@ -1,5 +1,5 @@
 server:
-  port: 8081
+  port: 8082
```

Comment:
If 8081 is standard for other spring modules, lest remain the same 8081 port

Reply:
fixed